client/main.lua

```diff
@@ -82,9 +80,9 @@ CreateThread(function()
             end
 
             -- Hotwiring while in vehicle, also keeps engine off for vehicles you don't own keys to
-            if IsPedInAnyVehicle(ped, false) and not IsHotwiring then
+            if IsPedInAnyVehicle(cache.ped, false) and not IsHotwiring then
```

Comment:
```suggestion
            if cache.vehicle and not IsHotwiring then
```
We want to use cache.vehicle here instead of using natives